src/client/components/pages/Courses/tableFields.tsx

```diff
@@ -298,6 +306,68 @@ export const formatMeetings = (
   );
 });
 
+type EnrollmentField = {
```

Comment:
Can add just a brief docblock explaining what this type is for?

Reply:
Yeah, good point - added that in 2252b03e